pylint/lint/run.py

```diff
@@ -489,7 +489,13 @@ def cb_verbose_mode(self, *args, **kwargs):
     def cb_enable_all_extensions(self, option_name: str, value: None) -> None:
         """Callback to load and enable all available extensions"""
         for filename in os.listdir(os.path.dirname(extensions.__file__)):
-            if filename.endswith(".py") and not filename.startswith("_"):
+            # pylint: disable=fixme
+            # TODO: Remove the check for deprecated check_docs after the extension has been removed
```

Comment:
👍

Reply:
Shall we remove the extension in `2.13`? Or do we consider this `3.0` worthy?

Reply:
Let's decide later. these todo list issue are very convenient, I'm going to do more of those. https://github.com/PyCQA/pylint/issues/5317 Feel free to edit :)

Reply:
You might want to look into Github projects, I find them even more convenient and much harder to lose out of sight as the issues on the front page increase.